### ISSUE TITLE
RGB LED does not fully turn off with PCA9685

### ISSUE THREAD

**Issue by scottgonzalez** [rwaldron/johnny-five#782]
I tried using two different types of common anode RGB LEDs and neither one fully turned off, they both were very dimly lit. I don't have any common cathode RGB LEDs to test with.

**Comment by rwaldron** [comment]
Using your example, I see complete on and complete off. Is this the wrong example to repro? Let me know!

**Comment by scottgonzalez** [comment]
Yeah, I have it wired as shown in the new example. I ran `node eg/led-rgb-anode-PCA9685.js`, then from the REPL I ran `led.stop().off()` and this is what I end up with:

![breadboard with PCA9685 and common anode RGB LED](http://f.cl.ly/items/283P2T2r0o1u2t2A183Y/Displaying%20IMG_0945.JPG)

**Comment by rwaldron** [comment]
I've tried several different RGB LEDs (diffused & clear) and I can't reproduce. I'll pick it back up tomorrow morning and see if I've missed something

**Comment by scottgonzalez** [comment]
This is due to the controller using a special value for fully on and fully off. See https://github.com/adafruit/Adafruit-PWM-Servo-Driver-Library/blob/8846078c6198314aa610213bcd0ce8a0a05a7bca/Adafruit_PWMServoDriver.cpp#L105-L115

I've verified that using these values works for me. I just ordered some common cathode RGB LEDs to test with, so I'll send a patch as soon as those come (should be here on Tuesday).
